src/bp/core/api.ts

```diff
@@ -192,7 +192,9 @@ const workspaces = (workspaceService: WorkspaceService): typeof sdk.workspaces =
     getBotWorkspaceId: workspaceService.getBotWorkspaceId.bind(workspaceService),
     getWorkspaceRollout: workspaceService.getWorkspaceRollout.bind(workspaceService),
     addUserToWorkspace: workspaceService.addUserToWorkspace.bind(workspaceService),
-    consumeInviteCode: workspaceService.consumeInviteCode.bind(workspaceService)
+    consumeInviteCode: workspaceService.consumeInviteCode.bind(workspaceService),
+    getWorkspaceUsers: workspaceService.getWorkspaceUsers.bind(workspaceService),
+    getWorkspaceUsersWithAttributes: workspaceService.getWorkspaceUsersWithAttributes.bind(workspaceService)
```

Comment:
What do you think of using a single method instead? Ex:
- getWorkspaceUsers('default') to get all attributes (or second arg as true?)
- getWorkspaceUsers('default', false) to ignore attributes
- getWorkspaceUsers('default', ['firstname', 'lastname']) for specific ones?

Reply:
We can do that, both exist in worksoace service and didnt think of refact.

Reply:
We don't need to refactor the usage in the workspace service (can be done internally), but when we add something to the SDK, it should be as clear as possible. I think we should have one method to get users, and add parameters to configure that output

Reply:
I'm thinking of 

```
interface GetWorkspaceUsersOtpions {
  includeSuperAdmins: boolean,
  attributes: string[]
}
getWorkspaceUsers(workspaceId, options)
```
to fetch all attributes I would do something like
`getWorkspaceUsers('myWorkspace', {attributes: ['*']})`

@allardy what do you think ?